UI/index.html

```diff
@@ -5,7 +5,83 @@
 		<link rel="stylesheet" type="text/css" href="styles.css">
 	</head>
 	<body>
-		<script src="script.js"></script>
+<!--		<script src="script.js"></script>-->
+		<script src="AdList.js"></script>
+		<script>
+			const adList = new AdList([
```

Comment:
Maybe let's create separate js file for testing instead of adding this code directly to HTML